src/ui/weights/WeightSlider.js

```diff
@@ -55,25 +56,30 @@ export type Props = {|
 export class WeightSlider extends React.Component<Props> {
   render() {
     return (
-      <label style={{display: "flex"}} title={this.props.description}>
-        <span style={{flexGrow: 1}}>{this.props.name}</span>
-        <input
-          type="range"
-          min={MIN_SLIDER}
-          max={MAX_SLIDER}
-          step={1}
-          value={weightToSlider(this.props.weight)}
-          onChange={(e) => {
-            const weight: Weight = sliderToWeight(e.target.valueAsNumber);
-            this.props.onChange(weight);
-          }}
-        />{" "}
-        <span
-          style={{minWidth: 45, display: "inline-block", textAlign: "right"}}
-        >
-          {formatWeight(this.props.weight)}
-        </span>
-      </label>
+      <Tooltip title={this.props.description} placement="top">
+        <Grid container justify="space-between">
+          <Grid item xs={4}>
+            {this.props.name}
+          </Grid>
+          <Grid item xs={5}>
+            <Slider
+              value={weightToSlider(this.props.weight)}
+              min={MIN_SLIDER}
+              max={MAX_SLIDER}
+              step={1}
+              valueLabelDisplay="off"
+              onChange={(_, val) => {
+                const weight: Weight = sliderToWeight(val);
+                console.log({weight});
```

Comment:
looks like debug code

Reply:
oops, yep, that's debug code. Maybe we should have a lint rule that errors on lines with console logs.

Reply:
That's a reasonable suggestion. There are some cases where we want to do it deliberately, but we could just disable the lint rule for those lines.